src/main/java/org/terasology/launcher/TerasologyLauncher.java

```diff
@@ -115,6 +115,13 @@ public void start(final Stage initialStage) {
             }
         });
 
+        launcherInitTask.setOnFailed(event -> {
+            Platform.exit();
+            openCrashReporterAndExit((Exception) event.getSource().getException());
```

Comment:
This throws a NPE for some reason. Maybe because of the `Platform.exit();` before we're doing more stuff with the crash reporter?

Reply:
We'll make a follow-up PR to take out crash reporter for now.